Sources/SwiftBasicFormat/Syntax+Extensions.swift

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import SwiftSyntax
+
+public extension TokenSyntax {
```

Comment:
Looking at this file again, is there a reason this extension needs to be public? If not, I’d prefer to keep it internal so we don’t increase our API surface area further.

Reply:
I did it to make `var indentationOfLine: Trivia` accessible from [ParseDiagnosticsGenerator.swift](https://github.com/apple/swift-syntax/blob/d7b149326f65d1721ad2210360ec7c8a2c18335f/Sources/SwiftParserDiagnostics/ParseDiagnosticsGenerator.swift#L737) which is in SwiftParserDiagnostics package.

I made this extension internal in [30bfccd](https://github.com/apple/swift-syntax/pull/1867/commits/30bfccdc9097a926e068bd839a4b79efc1e7d277).  Please let me know if this was not your intention 😄

Reply:
Ah, I meant if we could make `indentationOfLine` internal but I forgot that we use it from `ParseDiagnosticsGenerator`. Makes sense to be public 👍🏽